shiny/examples/input_file/app.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+
+from shiny import App, reactive, render, ui
+
+app_ui = ui.page_fluid(
+    ui.input_file("file", "File", accept=".csv"),
+    ui.input_checkbox_group(
+        "stats", "Summary Stats", choices=["Row Count", "Column Count", "Column Names"]
+    ),
+    ui.output_table("summary"),
+)
+
+
+def server(input, output, session):
```

Comment:
Please keep the types; they are helpful when the editor supports them (like VS Code and the shinylive online editor), and for some editing environments they are necessary for the editor to not underline things in red.

```suggestion
def server(input: Inputs, output: Outputs, session: Session):
```